.prettierrc.json

```diff
@@ -1,11 +1,18 @@
 {
   "arrowParens": "avoid",
-  "bracketSpacing": true,
-  "quoteProps": "consistent",
-  "printWidth": 100,
+  "quoteProps": "preserve",
+  "printWidth": 95,
```

Comment:
Why did we give 95?

Reply:
I've reviewed the files after the lint and overall print width of 100 was too long for our codebase.